types/src/main/scala/roc/types/decoders.scala

```diff
@@ -23,7 +26,7 @@ object decoders {
 
   implicit val stringElementDecoder: ElementDecoder[String] = new ElementDecoder[String] {
     def textDecoder(text: String): String         = text
-    def binaryDecoder(bytes: Array[Byte]): String = bytes.map(_.toChar).mkString
```

Comment:
The binary decoder tests only tested some arbitrary notion of what they should decode.  That's why I removed them in favor of integration tests which test against what Postgres is actually going to send.